build-logic/build.gradle.kts

```diff
@@ -1,12 +1,34 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
    `kotlin-dsl`
+   kotlin("jvm") version embeddedKotlinVersion
```

Comment:
is `embeddedKotlinVersion` defined by Gradle?

Reply:
yeah, although the source code & javadoc is missing https://github.com/gradle/gradle/issues/20995